include/sys/vmem.h

```diff
@@ -0,0 +1,24 @@
+#include <sys/types.h>
+
+#define VMEM_SLEEP 0x00000001
+#define VMEM_NOSLEEP 0x00000002
+#define VMEM_INSTANTFIT 0x00001000
+#define VMEM_BESTFIT 0x00002000
+
+typedef uintptr_t vmem_addr_t;
+typedef size_t vmem_size_t;
+typedef unsigned int vm_flag_t;
```

Comment:
This typename suggests `vm_flag_t` is somehow related to virtual memory subsystem. Please rename to `vmem_flags_t`.

Reply:
Good point! In fact, I have already renamed it to `vmem_flags_t` on my local branch. :D